workspace_server/src/main/java/gov/nasa/jpl/aerie/workspace/server/WorkspaceBindings.java

```diff
@@ -252,58 +255,137 @@ private void put(Context context) throws NoSuchWorkspaceException, IOException {
     }
   }
 
-  private void post(Context context) throws NoSuchWorkspaceException, IOException {
-    final var pathInfo = PathInformation.of(context);
+  private void post(Context context) throws IOException, SQLException, NoSuchWorkspaceException {
```

Comment:
The `post` method used to contain a `helpText` and would respond with the appropriate body format should the user's body be misformatted. The removal of this behavior is a regression and makes the API harder to use for clients who do not already know the appropriate format. Please restore and expand on the helpText.